src/core/server/ui_settings/integration_tests/doc_exists.ts

```diff
@@ -21,31 +21,30 @@ export const docExistsSuite = (savedObjectsIndex: string) => () => {
         conflicts: 'proceed',
         query: { match_all: {} },
       },
+      refresh: true,
+      wait_for_completion: true,
     });
 
     if (initialSettings) {
       await uiSettings.setMany(initialSettings);
     }
 
-    return { kbnServer, uiSettings };
+    return { uiSettings, supertest };
   }
 
   describe('get route', () => {
     it('returns a 200 and includes userValues', async () => {
       const defaultIndex = chance.word({ length: 10 });
-      const { kbnServer } = await setup({
+
+      const { supertest } = await setup({
         initialSettings: {
           defaultIndex,
         },
       });
 
-      const { statusCode, result } = await kbnServer.inject({
-        method: 'GET',
-        url: '/api/kibana/settings',
-      });
+      const { body } = await supertest('get', '/api/kibana/settings').expect(200);
```

Comment:
These uiSettings integration tests were still using the legacy server / hapi's `inject` API, had to rewrite them to use `supertest` instead.